### PR TITLE
Update to Feathers 3 and Vuex 3

### PR DESCRIPTION
**Pull request by murbanowicz**
Update to Feathers 3 and Vuex 3

### Summary

- Update dependencies to current 3.0, e.g. old feathers/errors to @feathers/errors etc.
- Update and test with Feathers 3.0 and Vuex 3.0
- app.service to app.use to follow Feathers 3.0

### CODE REVIEW

File: src/service-module/getters.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import getFilter from 'feathers-query-filters'
+import { filterQuery } from '@feathersjs/commons'
 import sift from 'sift'
 import { sorter, select, _ } from 'feathers-commons'
```
Comment (Mattchewone) [inline src/service-module/getters.js]: I believe this is also `@feathersjs/commons` It could probably be merged with the above import on L1

**Comment by murbanowicz** [comment]
Can someonetry to fix error with the CI ?

**Comment by marshallswain** [comment]
@murbanowicz due to all of the linting changes, it's difficult for me to see what's actually going on here.  Could you get rid of the linting changes so I can see that more clearly?

**Comment by murbanowicz** [comment]
@marshallswain  - I have run `standard --fix` but no change on files, so I am not sure what I can do?

**Comment by marshallswain** [comment]
Yeah.  Standard won't change the indentation of `.then` after a promise, so you'd have to go back through and restore it manually, using the github diff for reference.

**Comment by murbanowicz** [comment]
@marshallswain Looks like done. 
It's a bit strange, because it is WebStorm forcing this indentation to what I have commited as required by JS Standard :)

**Comment by marshallswain** [comment]
Thanks for fixing this.  It was clean enough for me to see what you changed.  There were too many conflicting updates for me to merge this, so I created a separate PR with the main changes.  Thanks again!

**Pull request closed without merging**